.github/workflows/container-scan.yaml

```diff
@@ -0,0 +1,48 @@
+name: Container vulnerability scan
+on:
+  schedule:
+    - cron: "0 4 * * *"
+  workflow_dispatch:
+
+permissions: read-all
+
+jobs:
+  container-scan:
+    if: github.repository_owner == 'cilium'
+    name: scan-container
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        image: [
+          {name: cilium, dockerfile: ./images/cilium/Dockerfile},
+          {name: clustermesh-apiserver, dockerfile: ./images/clustermesh-apiserver/Dockerfile},
+          {name: docker-plugin, dockerfile: ./images/cilium-docker-plugin/Dockerfile},
+          {name: hubble-relay, dockerfile: ./images/hubble-relay/Dockerfile},
+          {name: kvstoremesh, dockerfile: ./images/kvstoremesh/Dockerfile},
+          {name: operator-generic, dockerfile: ./images/operator/Dockerfile},
+        ]
+        branch: [v1.11, v1.12, v1.13] 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        ref: ${{ matrix.branch }}
+      - name: Set environment variables
+        uses: ./.github/actions/set-env-variables
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@4b4e9c3e2d4531116a6f8ba8e71fc6e2cb6e6c8c # v2.5.0
+      - name: Build local container
+        uses: docker/build-push-action@v2
```

Comment:
Can we replace `v2` with commit SHA?
The same for the scan action.

Reply:
Done, thanks!